plugins/home/api-report.md

```diff
@@ -187,6 +194,51 @@ export type ToolkitContentProps = {
   tools: Tool[];
 };
 
+// @public
+export type Visit = {
+  id: string;
+  name: string;
+  pathname: string;
+  hits: number;
+  timestamp: number;
+  entityRef?: string;
+};
+
+// @public (undocumented)
+export type VisitedByTypeKind = 'recent' | 'top';
+
+// @public (undocumented)
+export type VisitedByTypeProps = {
+  visits?: Array<Visit>;
+  numVisitsOpen?: number;
+  numVisitsTotal?: number;
+  loading?: boolean;
+  kind: VisitedByTypeKind;
+};
+
+// @public (undocumented)
+export type VisitFilter = {
+  field: string;
+  operator: '<' | '<=' | '==' | '>' | '>=' | 'contains';
+  value: JsonValue;
+};
+
+// @public
+export interface VisitsApi {
+  listUserVisits(queryParams?: VisitsApiQueryParams): Promise<Visit[]>;
+  saveVisit(pageVisit: Omit<Visit, 'id' | 'hits' | 'timestamp'>): Promise<void>;
```

Comment:
I don't see this used anywhere, what is the intended usage of this in the app? Is there anything we could include here to make usage more simple?

Reply:
This work ended up being broken down into smaller steps to simplify reviewing but in retrospect it was not a clean solution. On this PR https://github.com/backstage/backstage/pull/19643 two reference implementations are provided. It is being used in the [homepage component](https://github.com/backstage/backstage/pull/19643/files#diff-35c15175fb037e85a299b646d4fc9cea791001b95fbd39864fac88fa0bf7fb28)

Reply:
Ah alright, but I still only see `listVisits` used, what about `saveVisits`? 😅

Reply:
Sorry, that is in another PR 😅 . https://github.com/backstage/backstage/pull/19644
There is a component called `<VisitListener/>` saving the visits.

Reply:
@Rugvip, just pinging you since our last conversation on this issue.

Reply:
Yep makes sense! will resolve this